doc/source/migration_details.rst

```diff
@@ -5,36 +5,126 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+available in the addons path in the Odoo instance of the new version.
+
+Or, for older versions: check out the OpenUpgrade source code from Github
+for the branches you need. Each branch migrates to its version from the
+previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
+migrating across multiple versions, you need to run each version of
+OpenUpgrade in order. Skipping versions is not supported.
+
+2. Check coverage of the migration scripts for your installed modules
+*********************************************************************
+
+Check if there are migration scripts provided for the set of modules that
+are installed in your Odoo database. If there are modules for which no
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may me lacking. Check the module coverage in
+this documentation.
+
+At this stage, for some of the uncovered modules, you might want to uninstall
+them from your database and re-install it later on after the migration.
+This option might reduce the workload of the migration and can be applicable
+for "ux" modules like ``web_widget_image_download`` if you did not
+override this modules.
+
+
+3. Make a copy the database to migrate
+**************************************
+
+Decide which database you are going to upgrade. You absolutely *must* make a
+backup of your live database before you start this process!
+
+4. Adjust your Odoo configuration
+*********************************
+
+Edit the configuration files and command line parameters to point to the
+database you are going to upgrade. Run Odoo with the ``--update all
+--stop-after-init --load=web,openupgrade_framework`` flags.
+
+For versions earlier than 14.0 that are running the OpenUpgrade fork rather
+than Odoo itself, you do not pass the `load` parameter.
+
+5. Run and write and run and ... the Upgrade
+********************************************
+
+Run the upgrade and check for errors. You will probably learn a lot about
+your data and have to do some manual clean up before and after the upgrade.
+Expect to repeat the process several times as you encounter errors, clean up
+your data, and try again. If necessary, ask for help or report bugs on
+Github.
+
+Add your migration scripts
+--------------------------
+
+There are plenty of examples in the code of existing migration scripts within
+OpenUpgrade for your current version and from the previous versions.
+
+You can also refer to the following documents:
+
+- `The Odoo Migration Manager
+  <https://doc.therp.nl/openupgrade/migrationmanager.html>`_
+- `OpenUpgrade API <https://doc.therp.nl/openupgrade/API.html>`_
+- the code of OpenUpgrade API
+  `openupgrade.py <https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py>`_
+- `Development FAQ <https://doc.therp.nl/openupgrade/devfaq.html>`_
+
+The Trial and error process for the development of your scripts
```

Comment:
The stanzas below are great advice but maybe belong to a `development` section rather than a `runtime` section?

Reply:
Maybe a new page under "https://doc.therp.nl/openupgrade/development.html" ?

Reply:
Yes, sounds good!